configs/schemas/config.schema.json

```diff
@@ -312,6 +312,11 @@
           "default": false,
           "title": "Code Search Tool",
           "type": "boolean"
+        },
+        "use_dependency_tool": {
```

Comment:
Could we use the name `use_dependency_checker` instead? This aligns better with `VulnerableDependencyChecker`, and avoids any confusion with agentic tools since this step occurs outside the agent.

Reply:
Done.